lib/plugin/videoroom/participant_info.ex

```diff
@@ -0,0 +1,55 @@
+defmodule Janus.Plugin.VideoRoom.ParticipantInfo do
+  @moduledoc """
+  Struct holding info about Participants.
+
+  Contains following fields:
+  - `id` - unique ID of a publisher
+  - `display` - display name of active publisher, if any
+  - `audio_codec` - audio codec used by a publisher, if any
+  - `video_codec` - video codec used by a publisher, if any
+  - `simulcast` - `true` if the publisher uses simulcast (VP8 and H.264 only)
+  - `talking` - boolean indicating whether the publisher is talking or not (present only if the room is configured to measure audio levels)
+  """
+  @type t() :: %__MODULE__{
+          id: String.t(),
+          display_name: String.t(),
+          video_codec: String.t(),
+          audio_codec: String.t(),
+          using_simulcast?: boolean(),
+          talking?: boolean()
+        }
+
+  @codecs [:opus, :g722, :pcmu, :pcma, :isac32, :isac16] ++ [:vp8, :vp9, :h264, :av1, :h265]
+  @codec_mapping @codecs |> Enum.into(%{}, &{to_string(&1), &1})
+
+  @key_mapping %{
+    "id" => :id,
+    "display" => :display_name,
+    "video_codec" => :video_codec,
+    "audio_codec" => :audio_codec,
+    "simulcast" => :using_simulcast?,
```

Comment:
I think that name `simulcast?` is just enough, we wouldn't name field below `is_talking?` so why bother with `using`